lib/pages/History.dart

```diff
@@ -0,0 +1,195 @@
+import 'package:flutter/material.dart';
+import 'package:carriage_rider/pages/Upcoming.dart';
+import 'package:intl/intl.dart';
+import 'package:humanize/humanize.dart';
+import '../models/Ride.dart';
+import 'Current.dart';
+
+Map days = {
```

Comment:
I'm a little lost in the Figma but on iOS 12/05 at least for the current ride page, it looks like this is just using the first letter of each day? This obviously doesn't work for Saturday and Sunday tho so I'd assume it's fine to use the 3 letter abbreviations--double check with the PMs?
![image](https://user-images.githubusercontent.com/60579411/110214632-7d181080-7e73-11eb-9411-f68258eaca41.png)